### PR TITLE
Merge Dalton's Restructure

### PR DESCRIPTION
**Pull request by dnomotron**
Folder Structure
Added Tips, Notes, Practice Questions, Misc to each child (cert name) folder
Added PROPOSED_TEMPLATE
renamed previous gcp/redhat files to match structure

### CODE REVIEW

File: AWS/Foundational/note.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+These are all optional courses, not sure if anyones even gonna go for these?
```
Comment (0-Ayala) [inline AWS/Foundational/note.txt]: Please try to refrain from asking questions in documentation.

File: TEMPLATE [hunk 2]
```diff
@@ -0,0 +1,36 @@
+EXAMPLE:
```
Comment (0-Ayala) [inline TEMPLATE]: This would mean people are still submitting individual notes and not editing a combined document correct?

**Review by 0-Ayala — CHANGES_REQUESTED** [review]
Please review the comments and adjust as necessary.

File: AWS/Foundational/note.txt (hunk 1 above)
Comment (dnomotron) [reply AWS/Foundational/note.txt]: I deleted that already

File: TEMPLATE (hunk 2 above)
Comment (dnomotron) [reply TEMPLATE]: No, That would just be the format for each persons contributions to the Notes file. A begin to end block with each person appending their notes to the end of the file

**Review by 0-Ayala — APPROVED** [review]
Thanks!

**Pull request merged**